src/tools-evaluation.md

```diff
@@ -22,7 +38,17 @@ of using Linux to boot into an operating system.
 | [u-root](https://u-root.org)                             | Go       | BSD 3-Clause | ByteDance, Google et al |
 | [uutils/coreutils](http://uutils.github.io/)             | Rust     | MIT          | not for LinuxBoot       |
```

Comment:
```suggestion
| [uutils/coreutils](http://uutils.github.io/)             | Rust     | MIT          | not for LinuxBoot       |
| [util-linux](https://git.kernel.org/pub/scm/utils/util-linux/util-linux.git/about/)             | C     | GPLv2          | ... |
| [coreutils](https://www.gnu.org/software/coreutils/) | C | ... | ... |
```

Reply:
I want to add `util-linux` et al to a difference section later, since there will be more it relates/compares to, and keep this here focused on coreutils, since it's quite established as this larger package by now.